### PR TITLE
Arbitrum Sepolia wstETH and weETH

### PR DESCRIPTION
**Pull request by noisekit**
SIPs
- SIP-388: Adding Support for EtherFi weETH to Arbitrum V3 LP https://sips.synthetix.io/sips/sip-388/
- SIP-389: Adding Support for Yield Tokens from Aave and Lido to Arbitrum V3 LP https://sips.synthetix.io/sips/sip-389/
- SCCP-338: Set Arbitrum Yield-Generating Asset LP & Issuance Variables https://sips.synthetix.io/sccp/sccp-338/
- SCCP-345: Reconfigure Arbitrum Collateral Issuance Parameters https://sips.synthetix.io/sccp/sccp-345/

Extra
- Renamed to lowercase (what is is possible without redeploys)
- Simplify configs with `var`

Screenshot 2024-09-11 at 14 59 36
Screenshot 2024-09-11 at 15 04 41

### CODE REVIEW

**Comment by noisekit** [comment]
ah there we go been ok just few seconds ago
image

File: tomls/omnibus-arbitrum-sepolia/collaterals/weeth.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[invoke.CoreProxy_configureCollateral_weeth]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"  # "args" see below in the multiline block
+
+[[invoke.CoreProxy_configureCollateral_weeth.args]]
+tokenAddress = "<%= settings.weeth_address %>"
+oracleNodeId = "<%= extras.eth_oracle_id %>"
```
Comment (kaleb-keny) [inline tomls/omnibus-arbitrum-sepolia/collaterals/weeth.toml:8]: is the oracle id here wrong one?

File: tomls/omnibus-arbitrum-sepolia/collaterals/wsteth.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[invoke.CoreProxy_configureCollateral_wsteth]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"  # "args" see below in the multiline block
+
+[[invoke.CoreProxy_configureCollateral_wsteth.args]]
+tokenAddress = "<%= settings.wsteth_address %>"
+oracleNodeId = "<%= extras.eth_oracle_id %>"
```
Comment (kaleb-keny) [inline tomls/omnibus-arbitrum-sepolia/collaterals/wsteth.toml:8]: oracle node id issue

Comment (noisekit) [reply tomls/omnibus-arbitrum-sepolia/collaterals/wsteth.toml:8]: image

Comment (noisekit) [reply tomls/omnibus-arbitrum-sepolia/collaterals/wsteth.toml]: fixed

**Review by kaleb-keny — APPROVED** [review]

**Pull request merged**